lib/FieldType/Mapper/DateFormMapper.php

```diff
@@ -59,6 +60,7 @@ public function mapFieldValueForm(FormInterface $fieldForm, FieldData $data)
                         [
                             'required' => $fieldDefinition->isRequired,
                             'label' => $fieldDefinition->getName(),
+                            'attr' => ['data-timezone-offset' => $timezoneOffsetSeconds]
```

Comment:
This data attribute is specific for date picker component used in `ezplatform-admin-ui` so it should be generated here https://github.com/ezsystems/ezplatform-admin-ui/blob/1.5/src/bundle/Resources/views/fieldtypes/edit/ezdate.html.twig

Reply:
Resolve in: https://github.com/ezsystems/ezplatform-admin-ui/pull/1185/commits/32cd1a0d4eaa202b7da88b076cce4951bf20726d